app/vendors/nategood/httpful/.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
```

Comment:
I prefer not to include the third party libs in the repo. Is there a specific reason why this is included. Can it be installed through composer?

Reply:
@xcompass by default, composer installs the package under the top level `vendor`.  but cakephp only looks for libraries in [the top level `vendors` folder](https://book.cakephp.org/2.0/en/core-utility-libraries/app.html#loading-vendor-files).  we could create a symbolic link in the `vendors` folder